### PR TITLE
adding root to breadcrumb

### PR DESCRIPTION
**Pull request by Numbers88s**
Ref #24946 

Add link to root directory in file path breadcrumbs UI

### CODE REVIEW

**Comment by efritz** [comment]
Fixes #24946.

File: client/web/src/repo/FilePathBreadcrumbs.tsx [hunk 1]
```diff
@@ -27,13 +28,17 @@ export const FilePathBreadcrumbs: React.FunctionComponent<
     const partToClassName = (index: number): string =>
         index === parts.length - 1 ? 'test-breadcrumb-part-last' : 'part-directory test-breadcrumb-part-directory'
 
-    const spans: JSX.Element[] = []
+    const spans: JSX.Element[] = [
+        <LinkOrSpan key="root-dir" className="className" to={repoUrl} aria-current={false}>
```
Comment (efritz) [inline client/web/src/repo/FilePathBreadcrumbs.tsx:32]: You'll need to update this class name

**Comment by efritz** [comment]
Screen Shot 2021-09-29 at 2 54 14 PM

It seems odd that the root `/` is clickable but the separators aren't. Can we make the separators clickable and link to the proceeding directory?

File: client/web/src/repo/FilePathBreadcrumbs.tsx (hunk 1 above)
Comment (Numbers88s) [reply client/web/src/repo/FilePathBreadcrumbs.tsx:32]: oh boy, sorry about that. Got so excited I passed the cli checks 😅

Comment (efritz) [reply client/web/src/repo/FilePathBreadcrumbs.tsx:32]: I know that feel :comfy:

**Pull request merged**